### PR TITLE
add mockito dependencey.

### LINKED ISSUE #1174: [TASK]Make Junit5 + Mockito's Maven test dependencies effective.
## 需求建议

规整单元测试构建环境所需依赖，统一使用 junit5+mockito 作为测试套件，需要匹配好junit5和mockito版本，并进行简单的单测功能测试，如：忽略测试、mock实例方法、mock静态方法、参数化测试等。

欢迎认领

### PR DESCRIPTION
**Pull request by iwangjie**
Fixes #1174 

Changes proposed in this pull request:
- 

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: hippo4j-common/src/test/java/cn/hippo4j/common/MockitoTests.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.anyInt;
+
+/**
+ * Mockito 集成测试
```
Comment (magestacks) [inline hippo4j-common/src/test/java/cn/hippo4j/common/MockitoTests.java]: Please modify according to the historical code comment specification.

File: pom.xml [hunk 2]
```diff
@@ -126,6 +127,12 @@
                 <artifactId>netty-all</artifactId>
                 <version>${netty.version}</version>
             </dependency>
+
```
Comment (magestacks) [inline pom.xml]: 格式保持一致，删除该空行

Comment (iwangjie) [reply pom.xml]: done.

Comment (iwangjie) [reply pom.xml]: > 格式保持一致，删除该空行

done.

File: hippo4j-common/src/test/java/cn/hippo4j/common/MockitoTests.java (hunk 1 above)
Comment (iwangjie) [reply hippo4j-common/src/test/java/cn/hippo4j/common/MockitoTests.java]: > Please modify according to the historical code comment specification.

done.

**Review by magestacks — APPROVED** [review]

**Pull request merged**